src/searchForm/autoComplete/overlay/index.tsx

```diff
@@ -14,47 +14,47 @@ const StyledAutoComplete = styled(autoComplete)`
     padding: ${space.xl};
     /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
     box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
-  }
-
-  & .kirk-textField-wrapper {
-    background: transparent;
-    border-radius: 0;
-    border: 0;
-  }
-
-  & .kirk-textField-wrapper .kirk-autoComplete-icon {
-    padding: 0;
-    line-height: 0;
-    width: 48px;
-    height: auto;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
-
-  & .kirk-textField-wrapper input {
-    font-size: ${font.m.size}
-    line-height: ${font.m.lineHeight}
-    background: transparent;
-    border-radius: 0;
-  }
-
-  & .kirk-textField-wrapper .kirk-loader {
-    position: absolute;
-    top: 12px;
-    right: 12px;
-  }
-
-  & .kirk-items-list {
-    padding: 0;
-  }
-
-  & .kirk-items-list.kirk-autoComplete-list .kirk-items-list-item::before {
-    content: none;
-  }
 
-  & .kirk-items-list .kirk-items-list-item .kirk-item-choice {
-    padding: ${space.m} ${space.l};
+    .kirk-textField-wrapper {
+      background: transparent;
+      border-radius: 0;
+      border: 0;
+
+      .kirk-autoComplete-icon {
+        padding: 0;
+        line-height: 0;
+        width: 48px;
+        height: auto;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      input {
+        font-size: ${font.m.size}
+        line-height: ${font.m.lineHeight}
+        background: transparent;
+        border-radius: 0;
+      }
+
+      .kirk-loader {
+        position: absolute;
+        top: 12px;
+        right: 12px;
+      }
+    }
+
+    .kirk-items-list {
+      padding: 0;
+
+      &.kirk-autoComplete-list .kirk-items-list-item::before {
```

Comment:
`.kirk-items-list-item` is repeated on line **54** do we need this selector weight

Reply:
let's keep and be safe (since we don't know for sure).

Reply:
@gabrielagabriel I hesitated but I finally decided to split in 2 classes to improve readability. But I think you are right, regrouping everything inside one `kirk-items-list-item` selector will help understand nesting of HTML elements
I'll make the change thanks :)